src/frontends/pytorch/src/op/add.cpp

```diff
@@ -23,11 +23,25 @@ OutputVector translate_add_common(const NodeContext& context, bool inplace) {
     auto rhs = context.get_input(1);
     auto dtype0 = context.get_input_type(0);
     auto dtype1 = context.get_input_type(1);
+    
+        // Checking for boolean inputs
+    if (dtype0.is<type::boolean>() || dtype1.is<type::boolean>()) {
```

Comment:
This shouldn't work. You need to cast it to `element::Type` first. And then check for bool

Reply:
Done for add.cpp, extended this to mul.cpp